Python/Product/PythonTools/PythonTools/Repl/PythonInteractiveEvaluator.cs

```diff
@@ -661,7 +661,9 @@ partial class PythonInteractiveEvaluator :
         }
 
         internal Task InvokeAsync(Action action) {
-            return ((System.Windows.UIElement)_window.GetTextViewHost()).Dispatcher.InvokeAsync(action).Task;
+            // TOOD: Verify this
```

Comment:
I think this is fine. Might be worth ensuring that the action is actually run on the UI thread, but then again it should raise an exception if we create new controls on the wrong thread.

If the repl tests run okay, then tidy up the comments here and we'll call it fine.